integration/helpers/helpers.go

```diff
@@ -550,3 +550,49 @@ func UpsertAuthPrefAndWaitForCache(
 		assert.Empty(t, cmp.Diff(&pref, &p))
 	}, 5*time.Second, 100*time.Millisecond)
 }
+
+// WaitForNodeCount waits for a certain number of nodes to be visible
+// to the Proxy in clusterName.
+func WaitForNodeCount(ctx context.Context, t *TeleInstance, clusterName string, count int) error {
```

Comment:
Would this be more discoverable if it were a method on `TeleInstance`?

Can we get the cluster name from `t.Process.Config.Auth.ClusterName`, or do we sometimes have to use other cluster names?